### PR TITLE
yacht: split into instructions and introduction

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/yacht/introduction.md
Comment (ErikSchierboom) [inline exercises/yacht/introduction.md:1]: We might want to add a little story here.

File: exercises/yacht/instructions.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 
-The dice game [Yacht][yacht] is from the same family as Poker Dice, Generala and particularly Yahtzee, of which it is a precursor.
-In the game, five dice are rolled and the result can be entered in any of twelve categories.
-The score of a throw of the dice depends on category chosen.
+Given five dice and a category, calculate the score of the dice for that category.
+
+~~~~exercism/note
+You'll always be presented with five dice.
+Each dice's value will be between one and six inclusively.
+The dice will be unordered.
```
Comment (ErikSchierboom) [inline exercises/yacht/instructions.md:8]: Not sure what the nicest way is to list these "guarantees"

File: exercises/yacht/introduction.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+The dice game [Yacht][yacht] is from the same family as Poker Dice, Generala and particularly Yahtzee, of which it is a precursor.
+In the game, five dice are rolled and the result can be entered in any of twelve categories.
+The score of a throw of the dice depends on the category chosen.
```
Comment (IsaacG) [inline exercises/yacht/introduction.md:5]:
```suggestion
```suggestion
The game consists of twelve rounds.
In each, five dice are rolled and the player chooses one of twelve categories.
The chosen category is then used to score the throw of the dice.
```


```

File: exercises/yacht/instructions.md (hunk 1 above)
Comment (IsaacG) [reply exercises/yacht/instructions.md:8]: I think this is reasonable.

```suggestion
~~~~exercism/note
You'll always be presented with five dice.
Each dice's value will be between one and six inclusively.
The dice may be unordered.
```

File: exercises/yacht/introduction.md
Comment (ErikSchierboom) [reply exercises/yacht/introduction.md:1]: I've added a _tiny_ story

**Review by IsaacG — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

**Pull request merged**